sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamTest.scala

```diff
@@ -306,6 +306,14 @@ trait StreamTest extends QueryTest with SharedSparkSession with TimeLimits with
     def apply(func: StreamExecution => Any): AssertOnQuery = apply("Execute")(func)
   }
 
+  /** Call [[StreamingQuery.processAllAvailable()]] to wait. */
```

Comment:
Ah, seems like here is the problem about Unidoc:

```
[error] /home/runner/work/spark/spark/sql/core/target/java/org/apache/spark/sql/streaming/StreamTest.java:358: error: unexpected text
[error]   /** Call {@link StreamingQuery.processAllAvailable()} to wait. */
[error]            ^
[error] /home/runner/work/spark/spark/sql/core/target/java/org/apache/spark/sql/streaming/StreamTest.java:362: error: unexpected text
[error]   /** Call {@link StreamingQuery.processAllAvailable()} to wait. */
[error]            ^
```

When Unidoc produces a Javadoc, it compiles Scala codes and extract Scala docs and converts them into Javadoc. The problem is that looks `trait` doesn't work well out of the box with it ..

Probably we could just change it to ``` ``StreamingQuery.processAllAvailable()`` ```.

Reply:
Just FYI, the other errors are spurious. I tried to fix this problem so hard long ago but couldn't find a good way to fix, see also SPARK-20840

Reply:
Thank you for pointing it out :)